Fixes #81: update Azure Storage dependency to 7.x

paulirwin opened the pull request:
Resolves #81.

paulirwin committed:
Fixes #81: update Azure Storage dependency to 7.x

msftclas commented:
Hi __@paulirwin__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

msftclas commented:
__@paulirwin__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

mekoda commented:
Hi, thanks for the contribution! The idea is good, and it’s also a valuable task for the upcoming Milestone; however, there’s also other projects (like **SemanticLogging.Tests** and **SemanticLogging.Etw**) that also need to be updated in order to work out of the box. Could you update those projects too?

paulirwin committed:
Update dependencies in other projects; fix project build order

paulirwin commented:
@mekoda Thanks, updated. Note that I removed the references to the Database, TextFile, and WindowsAzure projects from the Etw project, as they didn't seem to be used and that would potentially be a bad thing since they are separate nuget packages. So, following from that, I removed the Azure storage nuget package from Etw as well. Let me know if for some reason that I should add those back.

mekoda commented:
Great, thanks!
